examples/kyb-app/src/components/organisms/UIRenderer/elements/JSONForm/components/LocalePicker/LocalePicker.tsx

```diff
@@ -0,0 +1,8 @@
+import { RJSVInputProps, TextInputAdapter } from '@ballerine/ui';
+import { languages } from './languages';
+
+export const LocalePicker = (props: RJSVInputProps) => {
+  props.schema.oneOf = languages;
```

Comment:
Avoid making mutations in React. More important to not mutate function parameters.